config/robot/aloha_mobile.yaml

```diff
@@ -0,0 +1,114 @@
+robot:
+  # Unique identifier for the robot (e.g., aloha_solo, aloha_stationary, aloha_mobile)
+  name: aloha_mobile                
+
+  # Frames per second for control and data updates (default: 50)
+  fps: 50                         
+
+  # Leader Arm Configuration (Human-controlled arms)
+  leader_arms:
+    - # Identifier for the right leader arm (naming convention: leader_<unique_identification>)
+      name: leader_right               
+
+      # Model type for compatibility and calibration
+      model: aloha_wx250s              
+
+      # Orientation side (options: left, right)
+      orientation: right               
```

Comment:
There's probably a better way to do this than setting the orientation. I know this is used to load modes files, right? Can we accomplish this a better way? Can we create directories per robot type and store the configs in here? Like for the solo, it doesn't make sense to tell the users to create a symlink to ttyDXL_leader_left